pkg/codec/vaapi/vp8.go

```diff
@@ -0,0 +1,494 @@
+package vaapi
+
+// reference: github.com/intel/libva-utils/encode/vp8enc.c
```

Comment:
knit: it should be https://github.com/intel/libva-utils/blob/master/encode/vp8enc.c. Otherwise, it's not found.